src/models/blockchains.model.ts

```diff
@@ -151,6 +157,57 @@ export class Blockchains extends Entity {
   }
 }
 
+export type BlockchainsResponse = {
+  ticker: string
+  hash: string
+  networkID: string
+  network: string
+  description?: string
+  index: number
+  blockchain: string
+  blockchainAliases: string[]
+  active: boolean
+  syncCheckOptions?: {
+    path?: string
+    body: string
+    resultKey: string
+    allowance?: number
+  }
+  enforceResult?: string
+  logLimitBlocks?: number
+  path?: string
+  evm?: boolean
+  redirects?: {
+    alias: string
+    domain: string
+  }[]
+}
+
+export function blockchainToBlockhainResponse(bl: Blockchains): BlockchainsResponse {
+  return {
+    ticker: bl.ticker,
+    hash: bl.hash,
+    networkID: bl.networkID,
+    network: bl.network,
+    description: bl.description,
+    index: bl.index,
+    blockchain: bl.blockchain,
+    blockchainAliases: bl.blockchainAliases,
+    active: bl.active,
+    syncCheckOptions: {
+      path: bl?.syncCheckOptions?.path,
+      body: bl?.syncCheckOptions?.body,
+      resultKey: bl?.syncCheckOptions?.resultKey,
+      allowance: bl?.syncCheckOptions?.allowance,
+    },
+    enforceResult: bl?.enforceResult,
+    logLimitBlocks: bl?.logLimitBlocks,
+    path: bl?.path,
```

Comment:
why is bl possibly null/undefined here but not in the first few with ticker and hash, etc...

Reply:
because these values are not on all the chains so they might be null, for example path is only used for AVAX and log limit blocks is not on all chains either